packages/core/docs/contributing/themes.md

```diff
@@ -10,9 +10,39 @@ Default theme is located in `packages/core/nuxt-theme-module` folder and recogni
 
 ## Configuration
 
-To inherit from the default theme in your integration theme you need to install private `@vue-storefront/nuxt-theme` package. from this repo.
+To inherit from the default theme from your integration theme you need to install a `@vue-storefront/nuxt-theme` [package](https://www.npmjs.com/package/@vue-storefront/nuxt-theme)
 
-In `nuxt.config.js` of your integration theme set `apiClient/composables/helpers `options to be named as your integration packages. For example under composables you can put `@vue-storefront/comemrcetools-composables`. It will be used in lodash templates inside `@vue-storefront/nuxt-theme` as a replacement for variables (like here `import { useProduct, useCart } from '<%= options.composables %>` - `options.composables` is a palceholder for value passed to composables option in `nuxt.config.js`)
+In `nuxt.config.js` of your integration theme add `@vue-storefront/nuxt-theme` into `buildModules` key, as a second value you can pass options e.g:
```

Comment:
```suggestion
In `nuxt.config.js` in your integration theme, add `@vue-storefront/nuxt-theme` into `buildModules` key, as a second value you can pass options e.g:
```

Reply:
Twice `in` (`In `nuxt.config.js` in`)? Is that grammarly right?

Reply:
Yes, but if you like you can change it to:

```suggestion
In `nuxt.config.js` within your integration theme, add `@vue-storefront/nuxt-theme` into `buildModules` key, as a second value you can pass options e.g:
```